<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="car">
       <property name="font">
        <font>
         <family>Adobe Myungjo Std M</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QToolBox" name="toolBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="widget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>300</width>
              <height>331</height>
             </rect>
            </property>
            <attribute name="label">
             <string>出售车辆</string>
            </attribute>
            <widget class="QComboBox" name="comboBoxFactory">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>10</y>
               <width>122</width>
               <height>27</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>12</x>
               <y>15</y>
               <width>40</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>厂家</string>
             </property>
            </widget>
            <widget class="QComboBox" name="comboBoxBrand">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>60</y>
               <width>122</width>
               <height>27</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>60</y>
               <width>40</width>
               <height>20</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>品牌</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>100</y>
               <width>278</width>
               <height>57</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>报价</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditPrice"/>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>万</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>220</y>
               <width>278</width>
               <height>56</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>金额</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditTotal"/>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>万</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="pushButtonSure">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>290</y>
               <width>93</width>
               <height>28</height>
              </rect>
             </property>
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButtonCal">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>290</y>
               <width>93</width>
               <height>28</height>
              </rect>
             </property>
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
            <widget class="QWidget" name="">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>180</y>
               <width>224</width>
               <height>29</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>数量</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox"/>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>剩余数量</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelLast">
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>300</width>
              <height>331</height>
             </rect>
            </property>
            <attribute name="label">
             <string>新车入库</string>
            </attribute>
           </widget>
          </widget>
          <widget class="QTextEdit" name="textEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="calc"/>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>380</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Adobe 宋体 Std L</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>车辆管理</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>379</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>销售管理</string>
    </property>
    <addaction name="actioncar"/>
    <addaction name="actioncalc"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actioncar">
   <property name="text">
    <string>车辆管理</string>
   </property>
  </action>
  <action name="actioncalc">
   <property name="text">
    <string>销售统计</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
